core/src/main/scala/kafka/utils/Log4jController.scala

```diff
@@ -20,7 +20,9 @@ package kafka.utils
 import java.util
 import java.util.Locale
 
-import org.apache.log4j.{Level, LogManager, Logger}
+import org.apache.logging.log4j.{Level, LogManager}
+import org.apache.logging.log4j.core.config.Configurator
+import org.apache.logging.log4j.core.LoggerContext
```

Comment:
IMO, why not rewrite against `org.slf4j`?

Reply:
`Log4jController` provides a dynamic `Logger` querying functionality, not logging itself. It is why it uses log4j `Logger`s directly.

In contrast, the streams module does not provide those kinds of functionality so it uses slf4j fascade.